src/test/java/de/tum/in/www1/artemis/notification/SingleUserNotificationServiceTest.java

```diff
@@ -379,21 +380,23 @@ void testNotifyUserAboutNewPossiblePlagiarismCase() throws MessagingException {
         ArgumentCaptor<MimeMessage> mimeMessageCaptor = ArgumentCaptor.forClass(MimeMessage.class);
         verify(javaMailSender, timeout(1000)).send(mimeMessageCaptor.capture());
         assertThat(mimeMessageCaptor.getValue().getSubject()).isEqualTo("New Plagiarism Case: Exercise \"" + exerciseTitle + "\" in the course \"" + course.getTitle() + "\"");
+        assertThat(mimeMessageCaptor.getValue().getContent()).asString().contains(POST_CONTENT);
     }
 
     /**
      * Test for notifyUserAboutFinalPlagiarismState method
      */
     @Test
-    void testNotifyUserAboutFinalPlagiarismState() throws MessagingException {
+    void testNotifyUserAboutFinalPlagiarismState() throws MessagingException, IOException {
         // explicitly change the user to prevent issues in the following server call due to userRepository.getUser() (@WithMockUser is not working here)
         userUtilService.changeUser(TEST_PREFIX + "student1");
         plagiarismCase.setVerdict(PlagiarismVerdict.NO_PLAGIARISM);
         singleUserNotificationService.notifyUserAboutPlagiarismCaseVerdict(plagiarismCase, user);
         verifyRepositoryCallWithCorrectNotification(PLAGIARISM_CASE_VERDICT_STUDENT_TITLE);
         ArgumentCaptor<MimeMessage> mimeMessageCaptor = ArgumentCaptor.forClass(MimeMessage.class);
         verify(javaMailSender, timeout(1000)).send(mimeMessageCaptor.capture());
-        assertThat(mimeMessageCaptor.getValue().getSubject()).isEqualTo(PLAGIARISM_CASE_VERDICT_STUDENT_TITLE);
+        assertThat(mimeMessageCaptor.getValue().getSubject()).isEqualTo("Verdict for your plagiarism case");
+        assertThat(mimeMessageCaptor.getValue().getContent()).asString().contains("Verdict reached in plagiarism case for exercise");
```

Comment:
We could use constants here for consistency

Reply:
Using constant here was the reason for the bug - let's keep it simple this time ;)